mesonbuild/interpreterbase/baseobjects.py

```diff
@@ -75,6 +75,11 @@ def __init__(self, *, subproject: T.Optional[str] = None) -> None:
     def display_name(self) -> str:
         return type(self).__name__
 
+    @property
+    def is_assignable(self) -> bool:
+        ''' Property used to indicate whether an object can be used at all '''
+        return True
```

Comment:
properties are really slow, you generally want to avoid using them if at all possible, I'd make this a class variable instead, set the base class to `True`, and have the `VoidHolder` class override it in its class definition.